src/tetris/brick/brickBias.ts

```diff
@@ -20,19 +20,42 @@ export default class BrickBias {
 	public set position(position: Vector2) {
 		this._position = position;
 	}
+
 	//endregion
 
 	//region public methods
 	//endregion
 
 	//region constructor
+	public static newFromTierList(field: Field, biasValue: number): BrickBias {
+		const brickBias = BrickBias.newDefault(field);
+		brickBias.chances = BrickChances.newNoChances();
+		
+		const distanceBetween = (a, b) => Math.pow(Math.abs(a - b), 2);
```

Comment:
It is slightly confusing that distanceBetween, does something different than this._distanceBetween. Recommendation: Rename one of the methods.

Reply:
It's accidental duplicate code, please delete `this._distanceBetween()`.